OfficeAddin/05 Using modern JavaScript/Lab.md

```diff
@@ -120,7 +120,7 @@ In this exercise, you will develop an Office Add-in using React and TypeScript.
 
 1. The project template that the Office Yeoman generator created include a number of React components that need to be updated or deleted.
 
-    Delete the **src/taskpane/components/HeroList.tsx** file.
+    Delete the **src/taskpane/components/HeroList.tsx** and **src/taskpane/components/Progress.tsx** files.
 
```

Comment:
And then on line 1795, the very end of the getQuote(symbol:string) code. it would be handy to include the comma at the very end (but not necessary). Just if you copy paste, it's easy to lose the comma so I recommend adding it like below.

    ```typescript
    getQuote(symbol:string) {
      return new Promise((resolve, reject) => {
        const queryEndpoint = `https://www.alphavantage.co/query?function=BATCH_STOCK_QUOTES&symbols=${escape(symbol)}&interval=1min&apikey=${ALPHAVANTAGE_APIKEY}`;

        fetch(queryEndpoint)
          .then((res: any) => {
            if (!res.ok) {
              reject('Error getting quote');
            }
            return res.json();
          })
          .then((jsonResponse: any) => {
            const quote: any = jsonResponse['Stock Quotes'][0];
            resolve(quote);
          });
      });
    },
    ```
Comma is on that very last line.

Reply:
OK... done